TF-6866 Update policy sets documentation (#395)
hashicorp/terraform-docs-common: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

-> **Note:** Terraform Cloud **Free** Edition includes one policy set of up to five policies. In Terraform Cloud **Plus** Edition, you can connect a policy set to a version control repository or create policy set versions via the API. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
<!-- END: TFC:only name:pnp-callout -->

> Policy sets are collections of policies that you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces, in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop the run.

Reviewer: ```suggestion
Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace.
```

### Create Policy Sets

To create a policy set:
1. Go to **Policy Sets** in your organization’s settings.
1. Click **Connect a new policy set**.
1. Choose your workflow.
   - For managed policies, click **create a policy set with individually managed policies**. Terraform Cloud shows a form to create a policy set and add individually managed policies.
   - For version control policies, choose a version control provider and then select the repository with your policy set. Terraform Cloud shows a form to create a policy set connected to that repository.
   - For automated workflows, click **No VCS Connection**. Terraform Cloud shows a form to create an empty policy set. You can use the API to add policies to this empty policy set later.

> 1. Choose a **Policy framework** for the policies you want to add. You can only create a policy set from policies written using the same framework. You cannot change the framework type after creation.

Reviewer: ```suggestion
1. Choose a **Policy framework** for the policies you want to add. A policy set can only contain policies that use the same framework (OPA or Sentinel). You cannot change the framework type after creation.
```

> 1. Choose a **Policy framework** for the policies you want to add. A policy set can only contain policies that use the same framework (OPA or Sentinel). You cannot change the framework type after creation.

Reviewer: I think this looks a little bit better if it's even more precise:
```suggestion
1. Choose a **Policy framework** for the policies you want to add. A policy set can only contain policies that use the same framework (OPA or Sentinel). You cannot change a policy set's framework type after creation.
```

Reviewer 2: ✅

1. Choose a policy set scope:

>    - **Policies enforced globally:** Terraform Cloud will automatically enforce this global policy set on all existing and future workspaces.

Reviewer 3: on all existing and future workspaces + and projects?

Reviewer 2: Yes, it is enforced on all the workspaces in the organization.
@ker-an Maybe we can add "in the organization" to be more explicit?

Author: Yup! Great suggestion @hs26gill, I will add that bit.

>    - **Policies enforced globally:** Terraform Cloud will automatically enforce this global policy set on all existing and future workspaces in the organization.

Reviewer: ```suggestion
   - **Policies enforced globally:** Terraform Cloud automatically enforces this global policy set on all of an organization's existing and future workspaces.
```

>    - **Policies enforced on selected projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to enforce this policy set on. This affects all current and future workspaces for any selected projects.

Reviewer: ```suggestion
   - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.
```

> 1. **(OPA Only)** Allow **Overrides** where users with override policy permissions can let Terraform apply plans that have [mandatory policy](#policy-enforcement-levels) failures.

Reviewer: ```suggestion
1. **(OPA Only)** Allow **Overrides**, which enables users with override policy permissions to apply plans that have [mandatory policy](#policy-enforcement-levels) failures.
```

> 1. **(VCS Only)** Specify the **VCS branch** within your VCS repository where Terraform Cloud should import new versions of policies. If you do not set this field, Terraform Cloud uses the default branch of the VCS repository you selected.

Reviewer: ```suggestion
1. **(VCS Only)** Optionally specify the **VCS branch** within your VCS repository where Terraform Cloud should import new versions of policies. If you do not set this field, Terraform Cloud uses your selected VCS repository's default branch.
```

> 1. **(VCS Only)** Specify the **Policies path** in your VCS repository containing the policy set files. This action lets you maintain multiple policy sets within a single repository. Set this field to the directory path that contains the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration file for the policy set. You can start the path with `/`, but this is optional. Terraform Cloud assumes that relative paths originate from the root of the repository. If you do not set this field, Terraform Cloud uses the repository root directory.

Reviewer: Suggesting shorteding this for brevity and readability:
```suggestion
1. **(VCS Only)** Specify where your policy set files live using the **Policies path**. This lets you maintain multiple policy sets within a single repository. Use a relative path from your root directory to the directory that contains either the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration files. If you do not set this field, Terraform Cloud uses the repository's root directory.
```

Author: Love it!

> 1. **(Managed Policies Only)** Select managed **Policies** to add to the policy set. You can only add policies written with the policy framework you selected for the policy set.

Reviewer: Does the UI say "Managed Policies"? Or just "Policies"? 
```suggestion
1. **(Managed Policies Only)** Select managed **Policies** to add to the policy set. You can only add policies written with the same policy framework you selected for this set.
```
